test/sai_test/config/route_configer.py

```diff
@@ -118,6 +135,50 @@ def t0_route_config_helper(
         #set expected dest T1
         test_obj.t1_list[2][100].l3_lag_obj = test_obj.dut.lag_list[1]
 
+    if is_create_route_for_vlan:
+        test_obj.vlan10_neighbor_device = Device(device_type=DeviceType.server, id=255, group_id=1)
```

Comment:
vlan is extended from routale_item, then it has the nexthop object as its attr, nexthop contains the nexthop devices